src/vs/workbench/contrib/chat/browser/chatInputPart.ts

```diff
@@ -571,6 +578,57 @@ export class ChatInputPart extends Disposable implements IHistoryNavigationWidge
 				});
 				widget.ariaLabel = ariaLabel;
 				widget.tabIndex = 0;
+			} else if (attachment.isImage) {
+				const blob = new Blob([attachment.value as Uint8Array], { type: 'image/png' });
+				const url = URL.createObjectURL(blob);
+
+				const img = document.createElement('img');
+				img.classList.add('chat-attached-context-image');
+				img.src = url;
+				img.alt = '';
+
+				// Pill with tiny image
+				const pill = document.createElement('div');
+				pill.classList.add('chat-attached-context-pill');
+				pill.tabIndex = -1;
```

Comment:
Not needed? I think what you're after is to have the container be focusable, is that `widget`?